test/bootstrap_fields_test.rb

```diff
@@ -428,4 +428,14 @@ class BootstrapFieldsTest < ActionView::TestCase
       end
     end
   end
+
+  test "can have a floating label" do
+    expected = <<-HTML.strip_heredoc
```

Comment:
We can use `<<~` now, instead of `strip_heredoc`. We no longer support Ruby versions that _don't_ have `<<~`.

Reply:
Sounds great, I wasn't really aware of this (just copy/pasted an existing test). But this would be the first and only occurrence of `<<~` in the project where `strip_heredoc` is used 220 times. Mind if I change all of them in this PR? Or would you prefer to only have this one changed and create a new PR for the rest?

Reply:
Never mind, just saw you added the PR https://github.com/bootstrap-ruby/bootstrap_form/issues/585. Will pick that one up after this and change this single occurrence of `strip_heredoc`.